test/fixtures/output/connect/offset/update-offset.golden

```diff
@@ -0,0 +1,8 @@
++------------+--------------------------------+
+| ID         | lcc-123                        |
+| Phase      | PENDING                        |
+| Message    | Alter offset process is        |
```

Comment:
The message is empty while the Phase is pending. We should not show this message here.

Reply:
This is just for test purposes. The field has omit if empty tag `Message   string `human:"Message,omitempty" serialized:"message,omitempty"`
	AppliedAt string `human:"Applied At,omitempty" serialized:"applied_at,omitempty"``